ui-tests/src/test/java/com/redhat/jenkins/plugins/ci/integration/docker/fixtures/ActiveMQContainer.java

```diff
@@ -33,16 +33,7 @@
 public class ActiveMQContainer extends DockerContainer {
 
     public String getBroker() throws IOException {
-        String ip = getIpAddress();
-        if (ip == null || ip.equals("")) {
-            JsonNode binding = inspect().get("HostConfig").get("PortBindings").get("61616/tcp").get(0);
-            String hostIP = binding.get("HostIp").asText();
-            String hostPort = binding.get("HostPort").asText();
-            ip = hostIP + ":" + hostPort;
-            return "tcp://"+ip;
-        } else {
-            return "tcp://"+ip+":61616";
-        }
+        return String.format("tcp://%s:%d", ipBound(61616), port(61616));
```

Comment:
I leave have to try this locally...I had issues when I started using podman on Fedora...hence why I had protection if ip was null.

Reply:
This will use localhost and port forwarding.
For anybody using mac you can't directly access the container ip, you need to use the forwarded ports